examples.template.html

```diff
@@ -0,0 +1,101 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Vis Network Examples</title>
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <style>
+      h1 {
```

Comment:
I don't think we should any unnecessary headers here. Hopefully at somepoint somebody redesignes this template (we should create an issue for that), but at the moment I would recommend reducing styles in the template to the minimum and let the browsers figure it out how to display it.

Reply:
On mobile the fonts are also WAY to big

Reply:
The template is only for the index file(s). The examples are kept intact, no harm done. They have optional meta headers but these can be avoided by rewriting the example (unless it has some special needs which is the reason why they exists but that should be rare).

Reply:
I'm talking about the overview index file.

Reply:
Why? Does it matter? It's just an index file. Nobody's going to read this anyway.

Reply:
I din't check it on mobile. That's crazy!

Reply:
It should be reasonable now.